e2e/framework/util.go

```diff
@@ -0,0 +1,34 @@
+package framework
+
+import (
+	"strconv"
+	"strings"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func GetControlPlaneHostAndPort(cl client.Client) (string, int32, error) {
+	var infraCluster configv1.Infrastructure
+	namespacedName := client.ObjectKey{
+		Namespace: "openshift-cluster-api",
```

Comment:
Do we already have this value in the constants?
Can we reference it from there?

Reply:
Yes, we do. Missed that.